mmv1/third_party/terraform/services/dataproc/resource_dataproc_cluster_test.go.erb

```diff
@@ -1344,6 +1366,25 @@ resource "google_dataproc_cluster" "basic" {
 `, rnd)
 }
 
+func testAccDataprocCluster_withMinNumInstances(rnd string) string {
+	return fmt.Sprintf(`
+resource "google_dataproc_cluster" "with_min_num_instances" {
+  name   = "tf-test-dproc-%s"
+  region = "us-central1"
+
+  cluster_config {
+	master_config{
```

Comment:
nit: Do you mind spacing these lines as two spaces rather than tabs, for consistency within the config?